android/sdk/src/main/java/org/jitsi/meet/sdk/ExternalAPIModule.java

```diff
@@ -130,6 +130,12 @@ public void sendEvent(String name, ReadableMap data, String scope) {
             return;
         }
 
+        if (name.equals("CONFERENCE_WILL_JOIN")) {
+            view.setCurrentConferenceUrl(data.getString("url"));
+        } else if (name.equals("CONFERENCE_WILL_LEAVE")) {
+            view.setCurrentConferenceUrl(null);
```

Comment:
how about failure? also, if we follow a link while in a conference, do we always get ended / failed for the ongoing conference **before** will join from the new one?

Reply:
Good point ! I need to check all that